std_msgs/Int32 srvA
geometry_msgs/Twist srvB
---
std_msgs/Bool srvC